### PR TITLE
[DEV-1347]: change subscribe button click logic

### PR DESCRIPTION
**Pull request by MikeAtUqido**
#### List of Changes

* When the webinar card is in the homepage section and the webinar was in the past the unauthenticated users should navigate to the webinar's detail page

#### Motivation and Context

At the moment the unauthenticated users was brought to the login page instead

#### How Has This Been Tested?

Manually

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .changeset/chilly-vans-glow.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"nextjs-website": patch
```
Comment (marcobottaro) [inline .changeset/chilly-vans-glow.md:2]: I think it's a new non-breaking feature
```suggestion
"nextjs-website": minor
```

File: apps/nextjs-website/src/components/molecules/SubscribeToWebinar/SubscribeToWebinar.tsx [hunk 2]
```diff
@@ -93,12 +93,23 @@ const SubscribeToWebinar = ({
...
 
   const onSubscribeWithoutUser = () => {
     setIsLoading(true);
+    const finalPath = !pathname.includes(`/webinars/${webinarSlug}`)
+      ? `/webinars/${webinarSlug}?action=subscribe`
+      : `${pathname}?action=subscribe`;
+
     // eslint-disable-next-line functional/immutable-data
-    router.push(`/auth/login?redirect=${pathname}`);
+    router.push(`/auth/login?redirect=${finalPath}`);
```
Comment (marcobottaro) [inline apps/nextjs-website/src/components/molecules/SubscribeToWebinar/SubscribeToWebinar.tsx:112]: The resulting URL is `/auth/login?redirect=/webinars/PagoPA-multe?action=subscribe` with a double question mark. You should replace `?` with `&` on line 109

**Review by marcobottaro — CHANGES_REQUESTED** [review]
[screencast-bpconcjcammlapcogcnnelfmaeghhagj-2024.03.13-12_31_18.webm](https://github.com/pagopa/developer-portal/assets/39835990/9c469066-f41c-45b0-8990-7829b51ba29d)
As you can see at minute 0:19 on the time scale of the video, there is an instant where the subscribe button is clickable

File: apps/nextjs-website/src/components/molecules/SubscribeToWebinar/SubscribeToWebinar.tsx (hunk 2 above)
Comment (MikeAtUqido) [reply apps/nextjs-website/src/components/molecules/SubscribeToWebinar/SubscribeToWebinar.tsx:112]: The second `?` is relative to the URL that the user will se after he has logged in. we can't change that with the `&`, one solution could be encode the redirectURL so the user don't see it in the first url

**Review by MarcoPonchia — APPROVED** [review]

**Review by tommaso1 — APPROVED** [review]

**Pull request merged**